ocaml/xapi/xapi.ml

```diff
@@ -295,7 +295,30 @@ let on_master_restart ~__context =
     debug
       "failed to set Host_metrics.live to true immediately; will have to wait \
        for regular heartbeat to arrive: %s"
-      (ExnHelper.string_of_exn e)
+      (ExnHelper.string_of_exn e) ;
+    let am_i_missing_certs () =
```

Comment:
Did you mean for this to be inside the exception handler?

Reply:
nope :)

Reply:
thanks, now I realise why ocamlformat didn't like my indentation